File: LCM/lcm-tester/lcm-tester.py
import lcm
import sys

lc = lcm.LCM()

sys.path.insert(0, '../')
sys.path.insert(0, '../lcmt')
sys.path.insert(0, '../mav')

from pose_t import pose_t

msg = pose_t();

msg.utime = 42;

lc.publish("test", msg.encode())

File: LCM/lcm-tester/lcm-tester.cpp


#include <iostream>


using namespace std;

#include <lcm/lcm-cpp.hpp>
#include "../trajectory_t.hpp"

int main(int argc,char** argv) {

    lcm::LCM* lcm = new lcm::LCM();

    trajectory_t msg;


    lcm->publish("test", &msg);



    return 0;
}
